### PR TITLE
Feature/roles-resource-dropdown-fetch

### PR DESCRIPTION
**Pull request by petrKavulok**
### in thin PR
- Detail containers were changed to fluid in order to cover more space on the page
- dropdown with search input and 'pagination'(`More...` button) was introduced in RolesResourcesCard, RolesDetailContainer & TenantDetailContainer
- ClientsDetailContainer was refactored to easily maintainable css grid layout with proper gap spacing

https://github.com/TeskaLabs/seacat-admin-webui/assets/79644454/bc3df98e-6269-407c-b8e8-90d2f8e7416a

Screenshot 2023-05-25 at 18 40 17
Screenshot 2023-05-25 at 18 40 42

### CODE REVIEW

File: src/modules/auth/clients/ClientCreateContainer.js [hunk 1]
```diff
@@ -195,177 +195,173 @@ const ClientCreateContainer = (props) => {
...
+						}
+						<TextInput name="client_uri" register={register} disabled={disabled} labelName={t('ClientCreateContainer|Client URI')}/>
+					</CardBody>
+					<CardFooter>
+						<ButtonGroup>
+						{((client != undefined) && (editClient == true)) ?
+							<>
 								<ButtonWithAuthz
-									title={t("ClientCreateContainer|Create")}
+									title={t("ClientListContainer|Save")}
```
Comment (Pe5h4) [inline src/modules/auth/clients/ClientCreateContainer.js]: @petrKavulok Well, this is `ClientCreateContainer`, not `ClientListContainer`

File: src/modules/auth/clients/ClientCreateContainer.js [hunk 2]
```diff
@@ -195,177 +195,173 @@ const ClientCreateContainer = (props) => {
...
 									color="primary"
 									type="submit"
 									disabled={isSubmitting}
 									resource={resource}
 									resources={resources}
 								>
-									{t("ClientCreateContainer|Create")}
+									{t("ClientListContainer|Save")}
```
Comment (Pe5h4) [inline src/modules/auth/clients/ClientCreateContainer.js]: @petrKavulok Same here

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Looks good! Just a small typo to fix

File: src/modules/auth/clients/ClientCreateContainer.js (hunk 1 above)
Comment (petrKavulok) [reply src/modules/auth/clients/ClientCreateContainer.js]: I didn't realize 'save' was in the common locales used across all apps, so we're using just `t("Save")` now

**Comment by petrKavulok** [comment]
@Pe5h4 

https://github.com/TeskaLabs/seacat-admin-webui/assets/79644454/2eb0981b-047e-4112-904a-9b83d2180d53

**Review by Pe5h4 — APPROVED** [review]
@petrKavulok I think this is good

**Pull request merged**